code/game/gamemodes/changeling/implements/hivemind.dm

```diff
@@ -0,0 +1,65 @@
+/mob/abstract/hivemind
+	name = "internal hivemind"
+	see_in_dark = 2
+	see_invisible = SEE_INVISIBLE_LIVING
+
+	var/mob/living/carbon/human/changeling_mob // the head honcho
+	var/list/mob/abstract/hivemind/hivemind = list() // who's in the head honcho's head
```

Comment:
remove these in destroy - also, since you store a ref to the changeling mob, why not just use its own list?